CONTRIBUTING.md

```diff
@@ -21,10 +21,11 @@ Run `npm run builder:check`
 
 ## Documentation
 
-To generate the usage snippet in the `README.md`, run:
+To generate the usage snippet in the `README.md`, first go to a project
+that _uses_ the archetype and then run:
```

Comment:
Hoping this can be moved to `builder-support` eventually so this isn't necessary (the copy & paste part I don't mind; switching to a project & installing your updated archetype is a hassle).